__tests__/pages/__snapshots__/curriculum.test.js.snap

```diff
@@ -109,6 +109,11 @@ exports[`Curriculum Page Should render curriculum with no session 1`] = `
           <div
             class="progress-card__container card shadow-sm mt-3 bg-primary text-white px-3 px-xl-4 py-4 border-0"
           >
+            <h4
+              class="progress-card__title mt-4 text-center "
```

Comment:
why would changing the env var give this snapshot change?

Reply:
Yeah, i encountered the same issue, the curriculum snap keeps getting regenerated, it think there's either something wrong with the snapshots or with the test, either case, we should stop using `toMatchSnapshot`.
https://github.com/garageScript/c0d3-app/issues/1773

Reply:
so it will sometimes regenerate with `Join C0D3 now and start your software engineering journey!`

And sometimes regenerate without that section?

Reply:
Yes, exactly. In a previous PR the snapshot was updated and the elements in this PR's snapshot were removed.